docs/design/integrate-with-cc-tools/README.md

```diff
@@ -0,0 +1,300 @@
+      CCID:     ccid,
+      Address:  address,
+      CC:       new(CCDemo),
+      TLSProps: *tlsProps,
+    }
+
+    return server.Start()
+}
+```
+
+After:
+
+```go
+
+// Import the FPC package first
+import (
+    fpc "github.com/hyperledger/fabric-private-chaincode/ecc_go/chaincode"
+)
+
+func runCCaaS() error {
+    address := os.Getenv("CHAINCODE_SERVER_ADDRESS")
+    ccid := os.Getenv("CHAINCODE_ID")
+
+    tlsProps, err := getTLSProperties()
+    if err != nil {
+        return err
+    }
+
+    //*Wrap the chaincode with FPC wrapper*//
+    var cc shim.Chaincode
+    if os.Getenv("FPC_ENABLED") == "true" {
+      cc = fpc.NewPrivateChaincode(new(CCDemo))
+    } else {
+      cc = new(CCDemo)
+    }
+
+    server := &shim.ChaincodeServer{
+      CCID:     ccid,
+      Address:  address,
+      CC:       cc,
+      TLSProps: *tlsProps,
+    }
+    return server.Start()
+}
+```
+
+Also, the user needs to install and update dependencies before building the code. We did mention that the interface is the same between the two wrappers but for some cases like the cc-tools-demo mock stub it was not implementing the `PurgePrivateData` function so it was breaking as FPC was implementing the function with `Panic(not Implemented)`. Managing private data is not supported with FPC but it has to adhere to the stub Interface of the standard fabric shim API which requires it. Since the cc-tools mock stub was an imported package, a good solution is to use `go mod vendor` and download all go packages in the vendor directory and edit it one time there. Running `nano $FPC_PATH/vendor/github.com/hyperledger-labs/cc-tools/mock/mockstub.go` and put the following block there:
+
+```go
+// PurgePrivateData ...
+ func (stub *MockStub) PurgePrivateData(collection, key string) error {
+     return errors.New("Not Implemented")
+}
+```
+
+Also, if you find other conflicts with the FPC standard packages released it may be a better option to copy your chaincode inside `$FPC_PATH/samples/chaincode/<YOUR_CHAINCODE>` since you've already followed [step 1](#1-setting-up-the-fpc-dev-environment) to setup the dev environment. And then you use the fpc package locally from within `import (fpc "github.com/hyperledger/fabric-private-chaincode/ecc_go/chaincode")`. More on this is driven by this [FPC tutorial](https://github.com/osamamagdy/fabric-private-chaincode/blob/feat/create-sample-app/samples/chaincode/simple-asset-go/README.md#writing-go-chaincode).
+
+#### Build the chaincode
+
+After setting up the development environment, you can build the chaincode using the [Ego](https://pkg.go.dev/github.com/edgelesssys/ego) framework which is necessary to run in encrypted enclaves. You can use the FPC build Makefile [here](https://github.com/osamamagdy/fabric-private-chaincode/blob/feat/create-sample-app/ecc_go/build.mk) but need to point the `CC_NAME` to it. The chaicode will be present as a docker image as the fabric network is using docker-compose.
```

Comment:
You can use the FPC build Makefile but YOU need to point .....

Reply:
Done